### ISSUE TITLE
`no-array-sort` should trigger only with `Array.prototype.sort()`

### ISSUE THREAD

**Issue by jonataswalker** [sindresorhus/eslint-plugin-unicorn#2744]
```js
import { type Collection } from 'mongodb'

const collection = await this.getColection()
const docs = await collection.find({ id }).sort({ expireAt: -1 }).limit(1).toArray()
```
